cli/cmd/cluster_addon_create_postgres.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated/cli/print"
+	"github.com/replicatedhq/replicated/pkg/kotsclient"
+	"github.com/replicatedhq/replicated/pkg/platformclient"
+	"github.com/replicatedhq/replicated/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+type clusterAddonCreatePostgresArgs struct {
+	version      string
+	diskGiB      int64
+	instanceType string
+
+	clusterID    string
+	waitDuration time.Duration
+	dryRun       bool
+	outputFormat string
+}
+
+func (r *runners) InitClusterAddonCreatePostgres(parent *cobra.Command) *cobra.Command {
+	args := clusterAddonCreatePostgresArgs{}
+
+	cmd := &cobra.Command{
+		Use:   "postgres CLUSTER_ID --version POSTGRES_VERSION",
+		Short: "Create a Postgres database for a cluster",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, cmdArgs []string) error {
+			args.clusterID = cmdArgs[0]
+			return r.clusterAddonCreatePostgresCreateRun(args)
+		},
+	}
+	parent.AddCommand(cmd)
+
+	_ = clusterAddonCreatePostgresFlags(cmd, &args)
+
+	return cmd
+}
+
+func clusterAddonCreatePostgresFlags(cmd *cobra.Command, args *clusterAddonCreatePostgresArgs) error {
+	cmd.Flags().StringVar(&args.version, "version", "", "The Postgres version to create (required)")
+	err := cmd.MarkFlagRequired("version")
+	if err != nil {
+		return err
+	}
+	cmd.Flags().Int64Var(&args.diskGiB, "disk", 200, "Disk Size (GiB) for the Postgres database")
+	cmd.Flags().StringVar(&args.instanceType, "instance-type", "db.t3.micro", "The type of instance to use for the Postgres database")
+
+	cmd.Flags().DurationVar(&args.waitDuration, "wait", 0, "Wait duration for addon to be ready (leave empty to not wait)")
+	cmd.Flags().BoolVar(&args.dryRun, "dry-run", false, "Dry run")
```

Comment:
Can you copy from here? https://github.com/replicatedhq/replicated/blob/main/cli/cmd/cluster_addon_create_objectstore.go#L60-L62